sharedb.planx.uk/sharedb-postgresql.js

```diff
@@ -81,8 +81,8 @@ PostgresDB.prototype.commit = function (
 
         client.query("BEGIN", (err) => {
           client.query(
-            "INSERT INTO flows (id, slug) VALUES ($1, $2) ON CONFLICT DO NOTHING",
-            [id, id],
+            "INSERT INTO flows (id, slug, name) VALUES ($1, $2, $3) ON CONFLICT DO NOTHING",
+            [id, id, id],
```

Comment:
This was an interesting one - without this I could create a flow, but not update it via ShareDB without an error.

I actually think this like of SQL will never get truly executed (it will _always_ hit the `ON CONFLICT` clause). This is because the flow is created via Hasura prior to any ShareDB interaction takes place. Postgres must just be "clever enough" to know that this query is incorrect without a `name` value, despite it never actually being executed. This is why `id, id, id` is ok as these values are never read.